assets/stylesheet/content.css

```diff
@@ -0,0 +1,549 @@
+    margin-bottom: 5%;
+}
+
+.b:hover {
+    background-color: #999;
+}
+
+img[alt="feat-story"] {
+    width: 100%;
+}
+
+.mag a {
+    color: #000;
+    font-size: 0.9rem;
+    font-weight: bolder;
+    padding-left: 2%;
+    padding-right: 2%;
+    display: block;
+}
+
+.mag a:hover {
+    border-bottom: 1px solid #f72210;
+    border-width: 2px;
+}
+
+.header_ban {
+    display: flex;
+}
+
+img[alt="main_mag"] {
+    width: 100%;
+}
+
+.before-editor {
+    margin-top: 5%;
+}
+
+img[alt="editor_pick"] {
+    width: 100%;
+}
+
+.editor a {
+    color: #000;
+    font-size: 1.2rem;
+    font-weight: bolder;
+    display: block;
+    padding-top: 1%;
+    padding-bottom: 2%;
+}
+
+.editor p {
+    color: #000;
+    font-size: 0.9rem;
+    padding-top: 2%;
+    padding-bottom: 3%;
+}
+
+img[alt="featured_slideshow"] {
+    width: 100%;
+}
+
+img[alt="featured_slideshow"]+a {
+    color: #000;
+    font-size: 1.1rem;
+    font-weight: bolder;
+    padding-top: 2%;
+    padding-bottom: 3%;
+}
+
+img[alt="others"] {
+    width: 100%;
+}
+
+.top a {
+    color: #000;
+    font-weight: bolder;
+    padding: 2% 0 0 3%;
+}
+
+img[alt="others"]+a {
+    color: #000;
+    font-size: 1.1rem;
+    font-weight: bolder;
+    padding-top: 2%;
+    padding-bottom: 3%;
+}
+
+.clearfix_2 {
+    height: 450px;
+}
+
+.before_footer {
+    width: 100%;
+    height: 460px;
+    background-color: #f7f7f7;
+    margin-top: 5%;
+}
+
+.before_footer_top {
+    display: flex;
+    margin-top: 3%;
+}
+
+.before_footer_top>div>.h1 {
+    color: #000;
+    font-size: 1.1rem;
+    font-weight: bolder;
+    padding: 2% 0 0 5%;
+}
+
+.before_footer_top>div>.h6 {
+    color: #000;
+    font-size: 0.9rem;
+    font-weight: bold;
+    padding: 2% 0 0 5%;
+}
+
+.panel {
+    padding: 5%;
+}
+
+.btn-warning {
+    background-color: #f72210;
+}
+
+.upper_foot {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 5%;
+    padding-left: 3%;
+    padding-right: 3%;
+}
+
+.lower_foot {
+    display: flex;
+    justify-content: space-between;
+}
+
+ul li {
+    display: inline;
+}
+
+footer ul li a {
+    border-right: 1px solid silver;
+    padding-right: 1%;
+    font-size: 0.9rem;
+    color: #333;
+}
+
+.lower_foot>div:first-child {
+    padding-left: 2%;
+    padding-right: 3%;
+}
+
+.lower_foot>div:last-child {
+    padding-left: 2%;
+    padding-right: 3%;
+}
+
+nav div {
+    color: white;
+}
+
+
+/*
+.nav-item a {
+    color: #000;
+    font-size: 0.9rem;
+    font-weight: bolder;
+}
+
+.nav-item:hover {
+    background: #000;
+    color: white;
+    cursor: pointer;
+}
+
+.navbar {
+    padding: 0;
+    border-bottom: 1px solid silver;
+    background: #f72210;
+}
+
+.nav-item li {
+    border-right: 1px solid silver;
+    line-height: 20px;
+    padding-right: 2.5%;
+    padding-left: 2.5%;
+    color: #333;
+    justify-content: space-around;
+    padding-top: 1%;
+}
+
+.nav-item li:last-child {
+    border-right: 0;
+}
+*/
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.